modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -10,9 +10,9 @@
 
 [NOTE]
 ====
-This Operator is General Availability for Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
+This Operator is General Availability for Amazon Web Services (AWS), global Microsoft Azure, Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
```

Comment:
Is "General Availability" properly used in this first sentence? Seems very off to me. Anyway, isn't it bordering on jargon (software industry term))? Would just plain "available" or "supported" work?